src/neo-vm/Types/StackItem.cs

```diff
@@ -32,37 +29,12 @@ public static StackItem FromInterface<T>(T value)
             return new InteropInterface<T>(value);
```

Comment:
I think that ` if (obj == null) return false;` in `Equals` could be removed without change the behaviour

Reply:
The returned value is `StackItem.Null`.

Reply:
Done.